doc/user/spec_examples.md

```diff
@@ -101,11 +102,52 @@ spec:
     backupIntervalInSecond: 300
     maxBackups: 5
     storageType: "S3"
+    s3:
+      s3Bucket: <S3-bucket-name>
+      awsSecret: <aws-secret-name>
+      prefix: <S3-prefix>
   restore:
     backupClusterName: "cluster-a"
     storageType: "S3"
 ```
 
+### Three members cluster that restores from different cluster's S3 backup
+
+etcd-operator can create a new cluster `cluster-b` from the backup of another cluster `cluster-a`. If backup doesn't exist, `cluster-b` starts as a new cluster. If backup exist, ensure that the spec of `cluster-b` has the same `s3Bucket`, `awsSecret`, and `prefix` as those on spec of `cluster-a`; so that etcd operator can find the correct s3 backup path from `cluster-a`.
```

Comment:
> If backup doesn't exist, `cluster-b` starts as a new cluster.

Well, this is not true.. Because if no backup exists, why does a user want to restore.. Why not just create a new cluster.

Reply:
When I restore from a cluster where the backup doesn't exist, I notice the new cluster just start as usual. I want to record this behavior so that the user knows what will happen. In contrast if I restore the same cluster and the backup doesn't exist, the cluster won't start because it complains the backup doesn't exist. So I think documenting those behaviors might know wha's expected.

Reply:
> When I restore from a cluster where the backup doesn't exist, I notice the new cluster just start as usual.

The behavior is undefined so far. Instead of dealing with it and coupling the issue here, we should open a new issue/PR to define this scenario.

Reply:
kk, I'll open another issue discussing on this.

Reply:
Can you remove the the line about this special case? Then we can move forward this PR.

Reply:
will do that!